Interfaces/API/Dirac.py

```diff
@@ -2612,26 +2452,24 @@ def peek( self, jobID, printout = False ):
        :type jobID: int or string
        :returns: S_OK,S_ERROR
     """
-    if type( jobID ) == type( " " ):
-      try:
-        jobID = int( jobID )
-      except Exception, x:
-        return self._errorReport( str( x ), 'Expected integer or string for existing jobID' )
-    elif type( jobID ) == type( [] ):
-      return self._errorReport( 'Expected int or string, not list' )
+    ret = self.__checkJobArgument( jobID, multiple = False )
+    if not ret['OK']:
+      return ret
+    jobID = ret['Value']
 
     monitoring = RPCClient( 'WorkloadManagement/JobMonitoring' )
     result = monitoring.getJobParameter( jobID, 'StandardOutput' )
     if not result['OK']:
       return self._errorReport( result, 'Could not retrieve job attributes' )
 
-    stdout = 'Not available yet.'
-    if result['Value'].has_key( 'StandardOutput' ):
-      self.log.verbose( result['Value']['StandardOutput'] )
-      stdout = result['Value']['StandardOutput']
-      if printout:
-        print stdout
+    stdout = result['Value'].get( 'StandardOutput' )
+    if stdout:
+      if printOutput:
+        self.log.always( stdout )
```

Comment:
The normal log level for the interactive commands is notice. In this case the output is printed without a usual log prefix with timestamp, etc. So, this should be rather:
     if printOutput:
        self.log.notice( stdout )
Similar for some other places